client/client.go

```diff
@@ -538,3 +542,32 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 var (
 	_ prometheus.Collector = (*Client)(nil)
 )
+
+// argListFromMySqlParams creates an array of strings from the pointer to the parameters for pt-mysql-sumamry
+func argListFromMySqlParams(pParams *agentpb.StartActionRequest_PTMySQLSummaryParams) []string {
```

Comment:
We typically don't add normal code after interface checking assertions like `_ prometheus.Collector = (*Client)(nil)` above. Please move it up

Reply:
I'm gonna fix this. I didn't know.